lib/pure/browsers.nim

```diff
@@ -33,10 +38,10 @@ proc openDefaultBrowser*(url: string) =
     var u = newWideCString(url)
```

Comment:
hm, windows is also using "open" here?

Reply:
That is an API Call I think, not a command, I am not a Windows expert tho.

Reply:
The documentation comment confirms that IS an API Call, yep. :slightly_smiling_face:

Reply:
I don't think it is. shellExecuteW executes a command.